src/denoising_model.jl

```diff
@@ -0,0 +1,35 @@
+export denoising_model
+
+include("denoising_data.jl")
+
+function denoising_model(shape, shape_p, KERNEL_SIZE, KERNEL_TYPE, KERNEL_SIGMA = 1.5)
+  sigma = 10^-3
+  data_path = joinpath(@__DIR__, "..", "images/cameraman.png")
+  cameraman_image = load(data_path)
+  x_t = vec(Float64.(cameraman_image))
```

Comment:
Can we solve this problem in something other than Float64?

Reply:
The problem is that cameraman_image is a Matrix{Gray{N0f8}}, and I need to consider real vectors in order to apply the Fourier and dwt (wavelet) transform. So maybe I can redefine it as Real ?